Update to `Window.showDirectoryPicker()` and `Window.showSaveFilePicker()` and `Window.showOpenFilePicker()` (#29581)
mdn/content: files/en-us/web/api/window/showdirectorypicker/index.md

      - : By specifying an ID, the browser can remember different directories for different
        IDs. If the same ID is used for another picker, the picker opens in the same
        directory.
    - `mode` {{optional_inline}}
      - : A string that defaults to `"read"` for read-only access or `"readwrite"` for read
        and write access to the directory.

>     - `startIn`

Reviewer: It looks like `id` and `startIn` are optional as well: https://wicg.github.io/file-system-access/#wellknowndirectory-determine-the-directory-the-picker-will-start-in (in all 3 methods)

### Exceptions

- `AbortError` {{domxref("DOMException")}}
  - : Thrown if the user dismisses the prompt without making a selection, or if the user
    agent deems the selected content to be too sensitive or dangerous.
- `SecurityError` {{domxref("DOMException")}}

>   - : An SecurityError is thrown if it was blocked by Same-Origin policy or it was not called via a user interaction such as a button press.

Reviewer: ```suggestion
  - : Thrown if the call was blocked by the [same-origin policy](/en-US/docs/Web/Security/Same-origin_policy) or it was not called via a user interaction such as a button press.
```

>   - : An SecurityError is thrown if it was blocked by Same-Origin policy or it was not called via a user interaction such as a button press.

Reviewer: ```suggestion
  - : Thrown if the call was blocked by the [same-origin policy](/en-US/docs/Web/Security/Same-origin_policy) or it was not called via a user interaction such as a button press.
```
